netlify.toml

```
@@ -2,6 +2,19 @@
   command = "yarn netlify-export"
   publish = "out"
 
+[[plugins]]
+  package = "@netlify/plugin-lighthouse"
+
+  [plugins.inputs.thresholds]
+    performance = 0.91
+    accessibility = 1
+    best-practices = 1
+    seo = 0.92
+    pwa = 0.3
+
+  [plugins.inputs]
+    output_path = "public/lighthouse.html"
```

```suggestion
    output_path = "lighthouse.html"
```

We don't need the `public/` directory prefix because this path represents the published asset path, not the source code path where we have to put the file inside the `public/` directory.
It is proven by this page that sits under [public/lighthouse.html](https://60e5a1952bfca20007e095f2--wargabantuwarga.netlify.app/public/lighthouse.html) path.